how-to/releaseduty/merge-duty/merge_duty.rst

```diff
@@ -167,8 +167,8 @@ Merge beta to release
    - where the yyy is the major Gecko version that Release now has.
 
 2. At the same time ``mozilla-beta`` should get a tag
-   `in the form FIREFOX_RELEASE_xxx_BASE <https://hg.mozilla.org/releases/mozilla-beta/rev/c4ed5781ba9260b6a46b97be4c66f32a28eea1a6>`__
-   - where the xxx is the major Gecko version that Beta had prior to the merge. (This should be
+   `in the form FIREFOX_RELEASE_yyy_BASE <https://hg.mozilla.org/releases/mozilla-beta/rev/c4ed5781ba9260b6a46b97be4c66f32a28eea1a6>`__
+   - where the yyy is the major Gecko version that Beta now has. (This should be
```

Comment:
beta's version doesn't change as part of this merge, so I think this sentence is just as confusing with this change?

Reply:
How about `that Release currently has`?

Reply:
I'd say either repeat "that Release now has" like in the previous sentence, or just refer to it and say "same as above", it's the same tag after all?